packages/shared/routes/dashboard/wallet/views/Send.svelte

```diff
@@ -52,6 +63,9 @@
     export let onSend = (..._: any[]): void => {}
     export let onInternalTransfer = (..._: any[]): void => {}
 
+    export let bridgeAddress = '0xF65e3cCbe04D4784EDa9CC4a33F84A6162aC9EB6'
+    export let chainId = '4002'
+
```

Comment:
Are these supposed to be hardcoded here?

Reply:
Nope, they will be changed to the values received from the parseing function.